m4_sessions/1803-inning/group_4.md

```diff
@@ -1,13 +1,46 @@
-Group 4
+## Group 4 Finding Public Data and Using it to Change the World
 
-Group Members:
+### Group Members:
+- Nora Gully
+- Kelley Jenkins
+- Casey Dallavalle
+- Josi McClellan
 
-Name of Session:
+### Name of Session: 
+- Finding Public Data and Using it to Change the World
 
-Session Description (Why should students come to this session):
+### Session Description (Why should students come to this session):
+- To learn about publicly available APIs that they can use with on their personal projects.
 
-Session Outcomes (What will students learn/leave with?):
+### Session Outcomes (What will students learn/leave with?):
+- A list of public apis.
+- Ideas for apps they can build with public data to solve real world problems.
+- AN IDENTITY AS A SOCIAL JUSTICE WARRIOR!
+- A sense of civic duty.
+
+
+### Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+
+- Icebreaker -- (10 min)
+- What is an api? -- (7.5 min) JOSI
+  - terminology 
+  - quick overview
+- Finding them -- (7.5 min) KELLEY
+  - understanding the docs
+  - examples of institutions that publish/maintain apis
+  - resources available
+- How do you know it's a good one? -- (7.5 min) CASEY
+  - Talk about data object and cleaning it
+- How to find inspiration & using them -- (7.5 min) NORA
+  - examples of apps using public data.
+ 
+ ~ 40 min
+
+### Group interactivity/discussion
+- Split up into groups of 4 or 5 via count off and individually research an api -- (10 min)
+- Share what you've found & brainstorm ideas on how to use that api to make an app -- (20 min)
+
+~ 30 min
```

Comment:
Be sure to CLOSE in some way--what are people taking away?  What next steps will they take?  What support do they need?